src/components/colorscale/color_attributes.js

```diff
@@ -64,14 +69,16 @@ module.exports = function makeColorScaleAttributes(context) {
         autocolorscale: extendDeep({}, colorScaleAttributes.autocolorscale, {
             description: [
                 'Has an effect only if `', context, '.color` is set to a numerical array.',
-                ' Determines whether or not the colorscale is picked using',
-                ' values inside `', context, '.color`.'
+                ' Determines whether the colorscale is the default palette (`autocolorscale: true`)',
+                ' or the palette determined by `', context, '.colorscale`.'
```

Comment:
@etpinard a small comment, although not in scope right now. In the mixed positive/negative case, we might at some point wish to correlate zero with the neutral (gray) color, and color saturation would correlate with the absolute value of the number (i.e. range(-10. 2) would yield lots of deep blues but only pale reds). To facilitate divergent colors generally, we might add a `ccenter` to the `cmin`/`cmax` tuple which would default to 0 (and/or perhaps  `(cmin+cmax)/2` as effectlvely happens now).